Configurations.md

```diff
@@ -1288,6 +1288,14 @@ If you want to ignore every file under the directory where you put your rustfmt.
 ignore = ["/"]
 ```
 
+## `ignore_missing_submod`
+
+Ignore missing submodule error.
+
+- **Default value**: `false`
+- **Possible values**: `true`, `false`
+- **Stable**: Yes
```

Comment:
All new options are unstable by default, so we'll need to update this. We don't need to create a tracking issue for the option just yet, but we'll need to mark this as unstable.